Dockerfile

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+# SPDX-License-Identifier: MIT
+
+# syntax=docker/dockerfile:1
+
+# Comments are provided throughout this file to help you get started.
+# If you need more help, visit the Dockerfile reference guide at
+# https://docs.docker.com/engine/reference/builder/
+
+################################################################################
+# Create a stage for building the application.
+
+ARG RUST_VERSION=1.72.1
```

Comment:
is there a way for this to be controlled by the rust-toolchain.toml file?

Reply:
Not in a simple way, there may be a way to copy the value from the rust-toolchain.toml file or install rust manually but I am not sure it is worth it at this point.